backend/scraper/mapping_police_violence.py

```diff
@@ -128,6 +131,7 @@ def make_tables_data_source(data_source, xwalk, configs):
     dat.to_csv(data_source + '.csv', index=True, header=True)
```

Comment:
nit: move `dat = map_varnames... dat = gen_ids...` out of the elif clauses since it's the same in each